### ISSUE TITLE
I would like to ask for the details of training the LinkRefiner network

### ISSUE THREAD

**Issue by yujiongh** [clovaai/CRAFT-pytorch#106]
Hello, thanks for sharing the modeling code. And the model performs well in many text detection tasks. I noticed in the paper that LinkRefiner network can be trained on the polygon-type datasets, such as TotalText and CTW-1500. But I can't find the details of the training, such as the GTs and the loss functions.  Would it be convenient for you to tell me the details of training the LinkRefiner network?

**Comment by ahfyzzy** [comment]
I met the same problem as you, did you figure out how to train LinkRefiner network? How you generate the GTs of link score, what's  the ratio of the width of refine link and the distance of paired points and what's the loss function when training? I would appreciate it to have your reply.
;)
